src/components/map/OlMap.vue

```diff
@@ -666,6 +689,33 @@
                         this.biodivData = feature.get('biodivData')
                         this.$refs.BiodivInformation.show()
                     }
+                } else {
+                    // handle clicks on enabled layers
+                    if (this.protectionAreasVisible) {
+                        const extent = this.view.calculateExtent(this.map.getSize() || null)
+                        const rcpExtent = ol.proj.transformExtent(
+                            extent,
+                            ol.proj.get('EPSG:3857'),
+                            ol.proj.get('EPSG:21781')
+                        )
+                        const position = ol.proj.transform(
+                            event.coordinate,
+                            ol.proj.get('EPSG:3857'),
+                            ol.proj.get('EPSG:21781')
+                        )
+                        respecterCestProtegerService.identify(
+                            position,
+                            rcpExtent,
+                            this.map.getSize()[0],
+                            this.map.getSize()[1],
+                            this.$language.current
+                        ).then(({ data }) => {
+                            if (data.results && data.results.length > 0) {
+                                this.swissProtectionAreaData = data.results[0]
```

Comment:
We only display item #0, not  other ones?

Reply:
We assume that there are no overlapping areas. There may be several results because the API has a tolerance, but in this case, it seems legit to consider the first result returned.